hw1/02-palindrome.html

```diff
@@ -9,19 +9,21 @@
       href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
     <script defer src="02-palindrome.js"></script>
   </head>
-  <body>
-    <main>
-      <section>
-        <h1>Exercise 02 - Palindrome</h1>
-        <div>
-          <label>Enter a positive number:</label>
-          <input type="number" />
-        </div>
-        <div>
-          <p>Is this a palindrome?</p>
-          <div></div>
-        </div>
-      </section>
-    </main>
+  <body class="bg-primary row">
+    <div class="container bg-light my-5 col-6 rounded">
+      <main>
+        <section>
+          <h1>Exercise 02 - Palindrome</h1>
+          <div>
+            <label>Enter a positive number:</label>
```

Comment:
This raises an error when you Wave the web page for accessibility. The label is not associated with an input label, which means that you should add the for="entry" as one of the attributes of the label element.

Reply:
Good catch! One of the requirements was to use the WAVE extension to solve this one error.